pyproject.toml

```diff
@@ -1,7 +1,119 @@
 [build-system]
-requires = ["setuptools"]
-build-backend = "setuptools.build_meta"
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "stability-sdk"
+dynamic = ["version"]
+description = "Python SDK for interacting with Stability AI APIs"
+readme = "README.md"
+requires-python = ">=3.8"
```

Comment:
Is >3.8 required as part of these changes, or can this be loosened?

Reply:
It further complicates dependencies (and may make some unsolvable) - and is EOL, with security updates ending in a month. Based on the last one it felt OK or perhaps even correct to sunset support.

Reply:
What actually forced the issue here is `tensorizer` - the package for it requires >=3.8. Moving to using the published package should allow us to entirely remove the submodule from `api-interfaces` (as it can pull `tensors.proto` from a url) - so I think it's worth it.

Reply:
It turns out we don't actually need the dependency, since the proto contract is all it uses. 3.7 support is restored.